modules/fundamental/src/vulnerability/endpoints/test.rs

```diff
@@ -106,13 +102,10 @@ async fn all_vulnerabilities(ctx: TrustifyContext) -> Result<(), anyhow::Error>
 #[test_context(TrustifyContext, skip_teardown)]
 #[test(actix_web::test)]
 async fn one_vulnerability(ctx: TrustifyContext) -> Result<(), anyhow::Error> {
-    let db = ctx.db;
+    let db = &ctx.db;
```

Comment:
Didn't the `TrustifyContext` even give you have `Graph` now?

Reply:
We can also remove `skip_teardown` to prevent unintentional partial-moves of `ctx.db` and friends. Forces folks to `clone()` if they need an owned.

Reply:
(because then the Context comes in via a &ref)

Reply:
Cool. I'll make another pass.